lang/en/tool_mergeusers.php

```diff
@@ -110,3 +110,9 @@
 $string['userselecttable_legend'] = '<b>Select users to merge</b>';
 $string['viewlog'] = 'See merging logs';
 $string['wronglogid'] = 'The log you are asking for does not exist.';
+$string['lastmergetomefrom'] = 'User last merged into this account';
```

Comment:
these two strings are making me thing as unnecessary long and complex.

I propose you a box similar to this:

| Last merged user |
|---|
| As "from": {here applicable text} |
| As "to": {here applicable text} |

If you agree on this or similar, strings should be adapted to their new content and purpose.

Reply:
I've worked it, so it looks like this now

![image](https://github.com/user-attachments/assets/8edd0e0e-61eb-4023-b729-d5d20c6c403c)